xclim/indices.py

```diff
@@ -225,16 +225,25 @@ def daily_freezethaw_cycles(tasmax, tasmin, freq='YS'):
 
     The number of days where Tmax > 0℃ and Tmin < 0℃.
     """
+
     ft = (tasmin < K2C) * (tasmax > K2C) * 1
     return ft.resample(time=freq).sum(dim='time')
 
 
 def daily_temperature_range(tasmax, tasmin, freq='YS'):
     r"""Mean of daily temperature range."""
-    dtr = tasmax - tasmin
+
+    dtr = abs(tasmax - tasmin)
```

Comment:
Why the abs ? Negative values would indicate a problem in input files that would be missed if taking the absolute value.